tips/TIP-0046/tip-0046.md

```diff
@@ -0,0 +1,987 @@
+      "0x523950126960352d2c5e35081d3f2128125f2b77502f4b1415440e3b260371216c1c645a",
+      "0x525e5a4d011d6359253e3e6e32717a2f0b354478154e1f4426345608234e044e5b62187c",
+      "0x6d1859664875241d0c6a4c55146f1d491c0a1f5a090e785c4a7b691977730e75622a5d41",
+      "0x742c2e2d080a42165c52732b4e48245e795e61240b7832625d1c0f037c5937191f510f07"
+    ],
+    "weakParents": [],
+    "shallowLikeParents": [],
+    "payload": {
+      "type": 0,
+      "tag": "0x746167",
+      "data": "0x6c754128356c071e5549764a48427b"
+    },
+    "maxBurnedMana": "864"
+  },
+  "signature": {
+    "type": 0,
+    "publicKey": "0xc7d2b4a2547167184522cfd5ef31ebb3fe50982430b40f869bfe1a43bf1211c8",
+    "signature": "0xe302f90891f71f5966878edbba0072f661c1fdc24f2afc207e20b7c9aa5ba3e2df1754f510cc6fe083a3bff1dd5c11aa6e851710e32b4db69ba2f6748e59db09"
+  }
+}
+```
+
+Block (hex-encoded binary serialization):
+
+```
+0x030000000000000000203e7ff3217b9b173a1e3b617060146e0362361a4b752833186108395f3b2b3d3e6c655e287d70767ea58d2af401000017432c5a7a672503480241125e3952414a7a320441080c624c264b004e09614a00062674252933745d134c1369621e366b0e63171c3e0b0c7b654009640b1316236469711226382854566f5e26533c391e2e4962434d13363a037e07563c522527566d536e165d0b0371523950126960352d2c5e35081d3f2128125f2b77502f4b1415440e3b260371216c1c645a525e5a4d011d6359253e3e6e32717a2f0b354478154e1f4426345608234e044e5b62187c6d1859664875241d0c6a4c55146f1d491c0a1f5a090e785c4a7b691977730e75622a5d41742c2e2d080a42165c52732b4e48245e795e61240b7832625d1c0f037c5937191f510f0700001800000000037461670f0000006c754128356c071e5549764a48427b600300000000000000c7d2b4a2547167184522cfd5ef31ebb3fe50982430b40f869bfe1a43bf1211c8e302f90891f71f5966878edbba0072f661c1fdc24f2afc207e20b7c9aa5ba3e2df1754f510cc6fe083a3bff1dd5c11aa6e851710e32b4db69ba2f6748e59db09
+```
+
+Block ID:
+
+```
+0x87b538efe6aaebf9b26e1c5a787df73a486e3c714e83fb018f3f5cd562e9facda5de0800
+```
+
+## Basic Block ID (Transaction payload)
+
+Block (json-encoded):
+
+```json
+{
+  "header": {
+    "protocolVersion": 3,
+    "networkId": "0",
+    "issuingTime": "1701088600401978103",
+    "slotCommitmentId": "0x01771954516b4c066163025c2671145a57041368613a434a0c445d2f1e06732f2bab32c3",
+    "latestFinalizedSlot": 500,
+    "issuerId": "0x267d4c21194e1d775a2f355f296c4c057e7e3907143a467c415d57244261311f"
+  },
+  "body": {
+    "type": 0,
+    "strongParents": [
+      "0x081446226d2158064559542c47511c344741405b58030109120552017967337522210a3e",
+      "0x0b2836550a6328063d21626b522c22151e1b232a5004155a727b237b6c063f5e51122e40",
+      "0x113e261616674f1634040b493f39146c0527670a0d5a381c1c254a2e0b5b343d19781008",
+      "0x1a4a01006e1b744839280411415b5a0c4e655d3927051c26782b41285003730657192262",
+      "0x37663d0977340229137d5b5140680766434b7b0b243c084025015d46671f3136275e6e26",
+      "0x4d7c4b734b5b70434a3709390415174a502c692f43723c7b360a20064b101028555c0548"
+    ],
+    "weakParents": [],
+    "shallowLikeParents": [],
```

Comment:
Not super important but these are omitempty so they wouldn't actually appear

Reply:
They are still printed with a recent iota.go version, not sure what's going on.

Reply:
Found the culprit and fixed it (https://github.com/iotaledger/hive.go/pull/624).

Reply:
Now fixed in the test vectors.